<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalSettingsDialog</class>
 <widget class="QDialog" name="GlobalSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>609</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>QEMU executables</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="simulatorName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="simulatorPath">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="simulatorsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Path</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="ChooseObjectWidget" name="configurationsPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>VM configurations directory</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChooseObjectWidget</class>
   <extends>QGroupBox</extends>
   <header>chooseobjectwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GlobalSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GlobalSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
